Sources/SwifterSwift/SwiftStdlib/FloatingPointExtensions.swift

```diff
@@ -47,6 +50,32 @@ public extension FloatingPoint {
     }
 }
 
+#if os(Android)
```

Comment:
Could we avoid these by, instead of using `Foundation.ceil` we use `self.rounded(.awayFromZero)`, and similarly for `floor`?

Reply:
I don't think they are the same:

```swift
zap ~ % swift repl
Welcome to Apple Swift version 6.0.3 (swiftlang-6.0.3.1.10 clang-1600.0.30.1).
Type :help for assistance.
  1> import Foundation
  2> ceil(-1.5)
$R0: Double = -1
  3> (-1.5).rounded(.awayFromZero)
$R1: (Double) = {
  _value = -2
}
```

Reply:
You're right. Just did an experiment and you should use `.up` for `ceil` and `.down` for `floor`. Or have I missed something?

Reply:
I think that so long as the tests still pass this would count as required for adding support for Android.

Reply:
https://github.com/SwifterSwift/SwifterSwift/pull/1231

Reply:
@marcprux merged my fix. Now you should be able to remove these extra conditions for Android.